.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
can you remove this file?

Reply:
+ 1
we should try to look out for files being added before committing. I guess that's why some people discourage using `git add .` 🤷‍♂